fleet_vehicle_configuration/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,3 @@
+This module provides a way to define several vehicule configuration.
+
+Example: car with 5 seats configuration or 7 seats configuration
```

Comment:
I think this module deserves a longer readme.

What the example suggests here seems to be the model IMO (like when you buy a car and it comes in different models / variants).

But in fact the configuration is an internal thing. It's how you alter the car for your company purposes, right?